wdl/src/main/scala/wdl/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -176,6 +176,22 @@ trait WdlStandardLibraryFunctions extends WdlFunctions[WomValue] {
     extractArguments flatMap arrayLength
   }
 
+  def flatten(params: Seq[Try[WomValue]]): Try[WomValue] = {
+    def getFlatValues(v: WomValue): List[WomValue] = v match {
+      case WomArray(_, values) => values.toList
+      case other => List(other)
```

Comment:
I think the default case should be a failure, since we're assuming that it's an array of arrays?